torch/_subclasses/meta_utils.py

```diff
@@ -295,13 +314,48 @@ def empty_create(inner_t, inner_src, symbolic_context=symbolic_context):
                     # which means that it will get caught by fake tensor mode.
                     # Ordinarily this would error, but there's some logic in
                     # fake tensor ensure this doesn't happen.
-                    r._coalesced_(t.is_coalesced())
+                    # Update: with the introduction of is_coalesced kw
+                    # argument to sparse coo tensor factory function,
+                    # there is no need to call r._coalesced_(t.is_coalesced()).
+                    # Is this note still relevant?
```

Comment:
It looks like you still query `is_coalesced` in some situations below, not sure if you wanted to modify those too.